pkgs/applications/editors/vscode/with-extensions.nix

```diff
@@ -46,12 +46,48 @@ let
   wrappedPkgVersion = lib.getVersion vscode;
   wrappedPkgName = lib.removeSuffix "-${wrappedPkgVersion}" vscode.name;
 
+  toExtensionJsonEntry = drv: rec {
+    identifier = {
+      id = "${drv.vscodeExtPublisher}.${drv.vscodeExtName}";
+      uuid = "";
+    };
+
+    version = drv.version;
```

Comment:
This also doesn't always exist.

Reply:
The python extension,

Reply:
Sorry, my mistake.

Reply:
Yeah I think version should be something required for mktplcRef:

https://github.com/NixOS/nixpkgs/blob/126bb19304148f3b6f76c35167932fbcc32648ee/pkgs/applications/editors/vscode/extensions/vscode-utils.nix#L56

https://github.com/NixOS/nixpkgs/blob/126bb19304148f3b6f76c35167932fbcc32648ee/pkgs/applications/editors/vscode/extensions/python/default.nix#L19-L24